File: pertemuan5/harga-barang-with-function/harga1_func.js
// Fungsi untuk menghitung total harga setelah diskon
function hitungTotalSetelahDiskon(harga, jumlah, diskon) {
  let totalSebelumDiskon = harga * jumlah;
  let totalDiskon = totalSebelumDiskon * (diskon / 100);
  return totalSebelumDiskon - totalDiskon;
}

// Program utama
let harga = parseFloat(prompt("Masukkan harga barang: "));
let jumlah = parseInt(prompt("Masukkan jumlah barang: "));
let diskon = parseFloat(prompt("Masukkan diskon dalam persen: "));

// Memanggil fungsi untuk menghitung total setelah diskon
let totalSetelahDiskon = hitungTotalSetelahDiskon(harga, jumlah, diskon);

// Menampilkan hasil
console.log(`Total harga setelah diskon: Rp${totalSetelahDiskon.toFixed(2)}`);

File: pertemuan5/harga-barang-with-function/harga1_func.c
 #include <stdio.h>
   
 // Fungsi untuk menghitung total harga setelah diskon
 float hitung_total_setelah_diskon(float harga, int jumlah, float diskon) {
    float total_sebelum_diskon = harga * jumlah;
    float total_diskon = total_sebelum_diskon * (diskon / 100);
    return total_sebelum_diskon - total_diskon;
  }
   
 int main() {
     float harga, diskon, total_setelah_diskon;
     int jumlah;
  
     // Input dari pengguna
     printf("Masukkan harga barang: ");
     scanf("%f", &harga);
     
     printf("Masukkan jumlah barang: ");
     scanf("%d", &jumlah);
     
     printf("Masukkan diskon dalam persen: ");
     scanf("%f", &diskon);
  
    // Memanggil fungsi untuk menghitung total setelah diskon
     total_setelah_diskon = hitung_total_setelah_diskon(harga, jumlah, diskon);
  
     // Menampilkan hasil
     printf("Total harga setelah diskon: Rp%.2f\n", total_setelah_diskon);
  
     return 0;
 }
  
